packages/storefront-lwc/src/modules/commerce/basket/basket.html

```diff
@@ -5,6 +5,7 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 -->
 <template>
+    <commerce-context-heading searchtext={searchtext} headertext="Your Basket"></commerce-context-heading>
```

Comment:
I could be wrong, but I don't think that `searchtext={searchtext}` is necessary?

Reply:
removed searchtext.